tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -1279,17 +1280,20 @@ def main():
 
     # Extract all of the kernel parameter and template type information.
     if args.add_static_casts:
-        KernelTemplateParams = {}
+	KernelTemplateParams = {}
+        # ignore caffe2/utils/math_gpu.cu for static_casts as PowKernel
+	# conflicts with kernel in caffe2/operators/pow_op.cu.
+	# Need to figure out a long-term solution.
         for filepath in all_files:
-            get_kernel_template_params(
-                filepath,
-                KernelTemplateParams,
-                CAFFE2_TEMPLATE_MAP if args.hipify_caffe2 else PYTORCH_TEMPLATE_MAP)
+	    if "math_gpu.cu" not in os.path.basename(filepath):
```

Comment:
That is a fragile solution. However, this will hopefully all go away soon.

Reply:
Yes, I am thinking of a more robust solution.